Python/Image Requests/img.py

```diff
@@ -0,0 +1,41 @@
+import requests
+from PIL import Image
+def img_requests(txt):
+    response=requests.get("https://source.unsplash.com/random/{0}".format(txt))
```

Comment:
Can we add more image sources, if that's possible?

Reply:
Yeah i think some I can add but , those will need an email id for api key and some are paid.  And how will we select one image source from more than one site if the user asks for one image only.?

Reply:
Please add a GSSOC Level in this PR